src/main/java/org/folio/consortia/controller/ConsortiumController.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.consortia.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.consortia.domain.dto.Consortium;
+import org.folio.consortia.rest.resource.ConsortiaApi;
+import org.folio.consortia.service.ConsortiumService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+public class ConsortiumController implements ConsortiaApi {
+  @Autowired
+  ConsortiumService consortiumService;
+
```

Comment:
We need to add new endpoint here
Get /consortia that should return consortia collection. Yes in our case it will return only single element always, but regardging REST contract we need to return collecation when using /consortia.
UI will use this endpoint to get consortium id

Reply:
Added